grid/crm/track-contact-event/maps/crisp.suma

```diff
@@ -0,0 +1,84 @@
+profile = "crm/track-contact-event@1.0"
+provider = "crisp"
+
+// Implements https://docs.crisp.chat/guides/rest-api/quickstart/
+
+map TrackContactEvent {
+  http POST "/v1/website/{parameters.WEBSITE_ID}/people/events/{input.contactId}" {
+    security "plugin_tier"
+
+    request "application/json" {
+      headers { "X-Crisp-Tier" = "plugin" }
+
+      body = {
+        text: input.eventName,
+        data: { ...input.eventProperties }
+      }
+    }
+
+    response 201 "application/json" {
+      return map result if (body.reason === "added") {
+        contactId = input.contactId
+      }
+
+      map error {
+        title = "Unexpected error"
+        detail = body
```

Comment:
Unsure how this assignment will be passed to the user

Reply:
You have specified the property type as string, so I am worried that passing in an object will cause a validaton error.

Reply:
@jnv @janhalama Since this is inside `application/json` handler already, I bet on the response actually being a JSON object and am directly stringifying it. Thanks!